gr-qtgui/python/qtgui/CMakeLists.txt

```diff
@@ -48,10 +48,17 @@ if(ENABLE_TESTING)
     ${CMAKE_BINARY_DIR}/gr-qtgui/swig
     )
 
+  if(LINUX AND NOT DEFINED ENV{DISPLAY})
```

Comment:
We definitely don't want to detect this at compile/config time but everytime we run a test with qtgui.